src/sass/_tokens.scss

```
@@ -13,6 +13,7 @@ $brand: (
   light: #ededef,
   gray: #545454,
   black: #231f20,
+  very_black: oklch(0.13 0.028 261.692),
```

Let's use kebab-case for the colors (like we're doing for the blues).

```suggestion
  very-black: oklch(0.13 0.028 261.692),
```